lib/constants.js

```diff
@@ -7,4 +7,5 @@ module.exports = {
 	E2FS_BLK_DISCARD:          10,
 	E2FS_BLK_CACHE_READAHEAD:  11,
 	E2FS_BLK_ZEROOUT:          12,
+	EINVAL:                   -22,
```

Comment:
This should be available from `os.constants.errno`

Reply:
We want the POSIX constant in all cases. I don't think it will be available on windows.

Reply:
Both POSIX & Windows specific constants are always available from the `os` module afaik; https://nodejs.org/api/os.html#os_error_constants – but `os.constants.errno.EINVAL` is `22`, not `-22`, so that'll need negation (assuming ext2fs always returns negative errnos)

Reply:
I don't see the windows specific errors on my linux box here so I guess there'll be no EINVAL on windows as well. I can't confirm this for now.